aip/0132.md

```diff
@@ -99,7 +99,7 @@ List methods implement a common response message pattern:
 
 ```proto
 message ListBooksResponse {
-  // The books located on the applicable shelf.
+  // The books from the applicable publisher.
```

Comment:
Can we change "applicable" to "specified"? "Applicable" sounds quite odd in this context.

Reply:
```suggestion
  // The books from the specified publisher.
```